### DISCUSSION TITLE
AppConfig PLIST details for AppleTV and MDM

### DISCUSSION THREAD

**Discussion by BAMcKay** [jellyfin/Swiftfin#1086] [Questions] [answered]
Thank you for your fantastic Swiftfin app!  We use it with Apple TVs and iPads at our K-12 school and so was wondering if you could post Swiftfin AppConfig PLIST details that we can push them out with Mosyle MDM - that way teachers and students don't need to enter their group login credentials?  Thanks!

**Comment by LePips** [comment, marked as answer]
I'm honestly surprised that this app is used in an institutional setting, but I have done no work whatsoever for MDM settings.

I have created an issue for me to investigate this:
- https://github.com/jellyfin/Swiftfin/issues/1103
